core/src/services/web-components.js

```diff
@@ -202,6 +210,7 @@ class WebComponentSvcClass {
    * If the web component is not defined yet, it gets imported.
    */
   renderWebComponent(viewUrl, wc_container, extendedContext, node, nodeId, isSpecialMf) {
+    const extendedContexts = { ...extendedContext, ...node };

```

Comment:
IMO it is not necessary to add the node on extendedContext, because there is already a property called `extendedContext.currentNode` which should be the same as `node`.

Reply:
The only reason for adding it, was that on the modal context, the currentNode didn't exist.